Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -168,13 +168,13 @@ ALWAYS_INLINE void Interpreter::set(Operand op, Value value)
 {
     switch (op.type()) {
     case Operand::Type::Register:
-        reg(Register { op.index() }) = value;
+        m_registers.data()[op.index()] = value;
         return;
     case Operand::Type::Local:
         m_locals.data()[op.index()] = value;
         return;
     case Operand::Type::Constant:
-        VERIFY_NOT_REACHED();
+        break;
```

Comment:
This sounds like a behavior change, being crashy when we hit this case would not be that bad
any reason for this change?

Reply:
We'll hit the `__builtin_unreachable()` right below the `switch`, this was mainly meant to slim down the code inlined for `Interpreter::set()` everywhere :)